packages/synapse-react-client/src/ror-client/index.ts

```diff
@@ -0,0 +1,16 @@
+import { fetchWithExponentialTimeout } from '@sage-bionetworks/synapse-client/util/fetchWithExponentialTimeout'
+import { ROROrganization, RORSearchResult } from './types/ROROrganization'
+
+export async function searchRegistry(query: string): Promise<RORSearchResult> {
+  return fetchWithExponentialTimeout(
+    `https://api.ror.org/v2/organizations?query=${encodeURIComponent(query)}`,
```

Comment:
This is cool.  A call to an external service, with exponential timeout, and the response is strongly typed.  👍